packages/commonwell-sdk/src/client/commonwell-member.ts

```diff
@@ -127,14 +126,18 @@ export class CommonWellMember implements CommonWellMemberAPI {
   async createOrg(organization: Organization, options?: BaseOptions): Promise<Organization> {
     const meta = options?.meta ?? buildBaseQueryMeta(this.memberName);
     const headers = this.buildQueryHeaders(meta);
-    const resp = await this.api.post(
-      `${CommonWellMember.MEMBER_ENDPOINT}/${this.memberId}/org`,
-      organization,
-      {
-        headers,
-      }
-    );
-    return organizationSchema.parse(resp.data);
+    try {
+      const resp = await this.api.post(
+        `${CommonWellMember.MEMBER_ENDPOINT}/${this.memberId}/org`,
+        organization,
+        {
+          headers,
+        }
+      );
+      return organizationSchema.parse(resp.data);
+    } catch (error) {
+      this.rethrowDescriptiveError(error, "Failed to create CW organization");
```

Comment:
Can we make the function `rethrowDescriptiveError` to return the Error object instead of throwing, so we throw it here?
- cleaner, as the function becomes a mapping/converting function only
- the stack trace won't have `rethrowDescriptiveError`, cleaner too